docs/usage_and_configuration/the_basics.md

```diff
@@ -196,6 +196,12 @@ Example: `black --line-ranges=1-10 --line-ranges=21-30 test.py` will format line
 
 This option is mainly for editor integrations, such as "Format Selection".
 
+```{note}
+Due to #4052, `--line-ranges` might format extra lines outside of the ranges when there
```

Comment:
```suggestion
Due to [#4052](https://github.com/psf/black/issues/4052), `--line-ranges` might format extra lines outside of the ranges when there
```
I don't think this will autolink